boards/banglejs-2/Makefile.dep

```diff
@@ -0,0 +1,18 @@
+# Use Segger's RTT unless another stdio_% is already used
+ifeq (,$(filter stdio_% slipdev_stdio,$(USEMODULE)))
+  USEMODULE += stdio_rtt
+endif
+
+ifneq (,$(filter mtd,$(USEMODULE)))
+  USEMODULE += mtd_spi_nor
+endif
+
+# USEMODULE += nrf5x_periph_shared
```

Comment:
This does not exist :wink:

Reply:
Right, not anymore, will remove it entirely, just slipped my attention.